tests/test_git.py

```diff
@@ -1041,3 +1041,32 @@ def test_status(
     assert staged["modify"] == ["foo"]
     assert unstaged == ["bar"]
     assert untracked == expected_untracked
+
+
+@pytest.mark.skip_git_backend("pygit2")
+@pytest.mark.parametrize("untracked_files", [False, True])
+def test_is_dirty(
```

Comment:
I'll suggest splitting this test into 2 tests:
1. test for untracked
2. test for modify-then-commit.

Reply:
Ended up getting rid of the parametrization and adding more tests for `is_dirty`

Reply:
Please try to split these blocks into individual tests if possible.